html/js/tree/ISet.js

```diff
@@ -0,0 +1,151 @@
+GTE.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new ISet.
```

Comment:
Maybe write Information Set _(creates a new information set)_ rather than the abbreviation in this comment, for clarity?